ApplicationServerDeployment/lambda/alexa/discovery.py

```diff
@@ -0,0 +1,86 @@
+import os
+
+import boto3
+import requests
+
+from response import AlexaResponse
+import logging
+import config
+import utils
+logger = logging.getLogger(__name__)
+
+
+class DiscoveryControl(object):

```

Comment:
add block comment to explain the purpose of the class

Reply:
Done